NitroxClient/GameLogic/Spawning/Metadata/Processor/CrashHomeMetadataProcessor.cs

```diff
@@ -0,0 +1,20 @@
+using NitroxClient.GameLogic.Spawning.Metadata.Processor.Abstract;
+using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Spawning.Metadata.Processor;
+
+public class CrashHomeMetadataProcessor : EntityMetadataProcessor<CrashHomeMetadata>
+{
+    public override void ProcessMetadata(GameObject gameObject, CrashHomeMetadata metadata)
+    {
+        if (gameObject.TryGetComponent(out CrashHome crashHome))
+        {
+            crashHome.spawnTime = metadata.SpawnTime;
+        }
+        else
+        {
+            Log.Error($"[{nameof(CrashHomeMetadataProcessor)}] Couldn't find {nameof(CrashHome)} on {gameObject}");
```

Comment:
Nit: Do we want to be consistent on wording "Could not" (as in `ConstructorMetadataProcessor`) and "Couldn't"? I personally would favor "Could not"